datasets/ar_res_reviews/ar_res_reviews.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dataset of 8364 restaurant reviews scrapped from qaym.com in Arabic for sentiment analysis"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{10.1007/978-3-319-18117-2_2,
+author="ElSahar, Hady
+and El-Beltagy, Samhaa R.",
+editor="Gelbukh, Alexander",
+title="Building Large Arabic Multi-domain Resources for Sentiment Analysis",
+booktitle="Computational Linguistics and Intelligent Text Processing",
+year="2015",
+publisher="Springer International Publishing",
+address="Cham",
+pages="23--34",
+isbn="978-3-319-18117-2"
+}
+"""
+
+_DESCRIPTION = """\
+Dataset of 8364 restaurant reviews scrapped from qaym.com in Arabic for sentiment analysis
+"""
+
+_HOMEPAGE = "https://github.com/hadyelsahar/large-arabic-sentiment-analysis-resouces"
+
+_DOWNLOAD_URL = (
+    "https://raw.githubusercontent.com/hadyelsahar/large-arabic-sentiment-analysis-resouces/master/datasets/RES1.csv"
+)
+
+
+class ArResReviews(datasets.GeneratorBasedBuilder):
+    """Dataset of 8364 restaurant reviews in Arabic for sentiment analysis"""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "polarity": datasets.ClassLabel(names=["-1", "1"]),
```

Comment:
```suggestion
                    "polarity": datasets.ClassLabel(names=["negative", "positive"]),
```

Can you use more explicit names for the labels please such as negative and positive ?

To make it work with the current _generate_examples you'll need to convert the values of the polarity field:
```python
polarity = "negative" if polarity == "-1" else "positive"
```


You'll have to regenerate the dataset_infos.json file after this change